mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/routes/v1/BrukerRoutes.kt

```diff
@@ -62,9 +69,45 @@ fun Route.brukerRoutes() {
                 call.respond(it)
             }
         }
+
+        post("ny") {
+            val request = call.receive<GetHistorikkForBrukerRequest>()
+            val norskIdent = request.norskIdent
+            val navIdent = getNavIdent()
+            val obo = AccessType.OBO(call.getAccessToken())
+
+            poaoTilgangService.verifyAccessToUserFromVeileder(getNavAnsattAzureId(), norskIdent) {
+                val message = createAuditMessage(
+                    msg = "NAV-ansatt med ident: '$navIdent' forsøkte, men fikk ikke sett deltakelser for bruker med ident: '$norskIdent'.",
+                    topic = "Se deltakelser",
+                    navIdent = navIdent,
+                    norskIdent = norskIdent,
+                )
+                AuditLog.auditLogger.log(message)
+            }
+
+            val response = historikkService.hentDeltakelserFraKomet(norskIdent, obo).map {
```

Comment:
Audit-logging kan gjøres via `.onRight` i stedet for `.map` (siden verdien ikke mappes til noe annet uansett).